department-of-veterans-affairs/abd-vro
# Feature - add Lookup Table for mapping Dropdown contentions

## What was the problem?

- adds new Lookuptable for mapping dropdown values from VA.gov
- this PR contains only the lookup table code for future use in Contention Classification endpoint(s)

Associated tickets or Slack threads:
- https://github.com/department-of-veterans-affairs/vagov-claim-classification/issues/214

## How does this fix it?[^1]


## How to test this PR
- run `pytest`


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
